bands/models.py

```diff
@@ -16,6 +16,11 @@ class Band(models.Model):
     def __str__(self):
         return self.name
 
+class TechnicalNeed(models.Model):
+    concert_name = models.ForeignKey('Concert')
```

Comment:
`concert_name` is a somewhat misleading name, since it's actually referring to a `Concert` object, not just a name.